test/verify/check-pages

```diff
@@ -158,10 +158,11 @@ OnCalendar=daily
         # b.wait_in_text("tr[data-goto-unit='test\.timer'] td:nth-child(3)", "morgen um")
 
         # BIOS date parsing using moment
-        b.go("/system/hwinfo")
-        b.enter_page("/system/hwinfo")
-        b.wait_present("#hwinfo .info-table-ct")
-        b.wait_in_text("#hwinfo .info-table-ct tbody:nth-of-type(2) tr:nth-of-type(3) td", ".20")
+        if m.image != 'rhel-7-5':
```

Comment:
The previous test also already checked for `.20`, why did that stop working on rhel-7-5?

Reply:
Oh this test was added by commit on hwinfo. But I amended the rhel-7-5 check to the wrong commit. Will change this.